frontend/.env

```diff
@@ -1,7 +1,8 @@
 #Read more here: https://create-react-app.dev/docs/adding-custom-environment-variables/
 
 #For locally: http://localhost:5000/rest-api
-REACT_APP_REST_API_BASE_URL=https://coderscamp2020-tablesoccer.herokuapp.com/rest-api
+#REACT_APP_REST_API_BASE_URL=https://coderscamp2020-tablesoccer.herokuapp.com/rest-api
+REACT_APP_REST_API_BASE_URL=http://localhost:5000/rest-api
```

Comment:
Hmmmm.... trzeba obczaić jak robić deploy na Heroku, bo teraz idzie z tym .env i w takim razie nie bedzie robić requestów nasz front. Ale w sumie na razie w ogóle jak widać nie działa jakoś przez CORS więc to bez znaczenia :)